examples/README.md

```diff
@@ -40,6 +40,19 @@ real applications with Kubernetes.
 Demonstrations of how to use specific Kubernetes features can be found in our [documents](../docs/).
 
 
+### Maintained Samples
+
+Name | Description
+------------- | ------------- | ------------
+[Guestbook](guestbook/) | PHP app with Redis
+[WordPress](mysql-wordpress-pd/) | WordPress with MySQL
+[Cassandra](cassandra/) | Cloud Native Cassandra
+
+Note: Please add samples to the list above that are maintained.
```

Comment:
The maintained examples will need to be updated every release, to use the latest and greatest features and current best practices, and to refresh command syntax, output, changed prerequisites, etc.